dask/tests/test_spark_compat.py

```diff
@@ -0,0 +1,108 @@
+import signal
+import sys
+import threading
+
+import pytest
+
+from dask.datasets import timeseries
+
+dd = pytest.importorskip("dask.dataframe")
+pyspark = pytest.importorskip("pyspark")
+
+from dask.dataframe.utils import assert_eq
+
+pytestmark = pytest.mark.spark
+
+# pyspark auto-converts timezones -- round-tripping timestamps is easier if
+# we set everything to UTC.
+pdf = timeseries(freq="1H").compute()
+pdf.index = pdf.index.tz_localize("UTC")
+pdf = pdf.reset_index()
+
+
+@pytest.fixture(scope="module")
+def spark_session():
+    if not sys.platform.startswith("linux"):
+        pytest.skip(
+            "Unnecessary, and hard to get spark working on non-linux platforms",
+        )
```

Comment:
Style nit: thoughts on moving this skip up to the top of this module instead of inside the `spark_session` fixture? (not meant to be a blocking comment)

Reply:
I originally had it this way, but then thought it would be better to apply it to the actual thing which we expect not to work. But I'm agnostic, happy to move it back up